engine/equations.cc

```diff
@@ -180,7 +180,8 @@ namespace MathDAG
                   OperationType::Type op, OperationType::Type accum, double groupIdentity)
     {
       // check if any arguments have x-vectors, and if so, initialise r.xVector
-      size_t oldNumElems=r.numElements();
+      // For feature 47
```

Comment:
At some point "feature 47" is not a helpful comment. "Sparse data representation" might be more useful, if needed.

Reply:
Thanks, I'll improve the description before I commit the latest changes.